packages/clay-autocomplete/src/ClayAutocomplete.js

```diff
@@ -530,6 +550,17 @@ ClayAutocomplete.STATE = {
 		}
 	}),
 
+	/**
+	 * Flag to indicate if dropdown is expanded.
+	 * @default false
+	 * @instance
+	 * @memberof ClayAutocomplete
+	 * @type {?bool}
+	 */
+	expanded: Config.bool()
```

Comment:
maybe it's better to rename it to `_expanded` since it's internal and used only here. Add a `@private` as well.